tests/baselines/reference/mappedTypeIndexedAccess.errors.txt

```diff
@@ -36,7 +36,7 @@ tests/cases/compiler/mappedTypeIndexedAccess.ts(24,5): error TS2322: Type '{ key
     // Error expected here
     let pair2: Pairs<FooBar>[keyof FooBar] = {
         ~~~~~
-!!! error TS2322: Type '{ key: "foo"; value: number; }' is not assignable to type 'Pair<FooBar>'.
+!!! error TS2322: Type '{ key: "foo"; value: number; }' is not assignable to type '{ key: "foo"; value: string; } | { key: "bar"; value: number; }'.
 !!! error TS2322:   Types of property 'value' are incompatible.

```

Comment:
Also, kind of weird that it skips a level of elaboration, but that's out of scope here.